### DISCUSSION TITLE
How to silent successful passes of gitleaks?

### DISCUSSION THREAD

**Discussion by hellt** [super-linter/super-linter#3165] [Q&A]
Hi,
I wonder how do I make gitleaks linter to be less verbose if all is good? Ideally I want gitleaks to emit logs only if there is a leakage detected, not when it is all good.

**Comment by ferrarimarco** [comment]
We made the gitleaks a bit less verbose in recent releases. Please, open a new issue if this is not the case for you, so we can investigate :)
